doc/src/records/dhwsys.md

```diff
@@ -144,11 +155,137 @@ Hourly hot water use (at the point of use).  See further info under wsDayUse.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly and at the end of interval") %>
+
+**wsFaucetDrawDurF=*float***
+
+Fauce water heater for draw duration factor to be use by warm-up waste and pipe lossesfor DHWDAYUSE draws.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly and at the end of interval") %>
+
+**wsShowerDrawDurF=*float***
+
+Shower water heater for draw duration factor to be use by warm-up waste and pipe lossesfor DHWDAYUSE draws.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly and at the end of interval") %>
+
+**wsBathDrawDurF=*float***
+
+Bath water heater for draw duration factor to be use by warm-up waste and pipe lossesfor DHWDAYUSE draws.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly and at the end of interval") %>
+
+**wsCWashrDrawDurF=*float***
+
+Clothed washer water heater for draw duration factor to be use by warm-up waste and pipe lossesfor DHWDAYUSE draws.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "Hourly and at the end of interval") %>
+
+**wsDWashrDurF=*float***
+
+Dish washer water heater for draw duration factor to be use by warm-up waste and pipe lossesfor DHWDAYUSE draws.
+
+<%= member_table(
+  units: "",
+  legal_range: "x $\\geq$ 0",
+  default: "1.0",
+  required: "No",
+  variability: "Hourly and at the end of interval") %>
+
+**wsUnkDrawWaste=*float***
+
+Unknown water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsFaucetDrawWaste=*float***
+
+Faucet water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsShowerDrawWaste=*float***
+
+Shower water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsBathDrawWaste=*float***
+
+Bath water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsCWashrDrawWaste=*float***
+
+Clothing washer water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
+
+**wsDWashrDrawWaste=*float***
+
+Dish washer water waste per draw.
+
+<%= member_table(
+  units: "gal",
+  legal_range: "x $\\geq$ 0",
+  default: "0.0",
+  required: "No",
+  variability: "Hourly") %>
```

Comment:
The content below was general documentation for a set of inputs. It should be preserved.